connector/connect/client/src/main/scala/org.apache.spark.sql.connect.client/Dataset.scala

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
The location should be ` connector/connect/client/src/main/scala/org/apache/spark/sql/connect/client/Dataset.scala` and not ` connector/connect/client/src/main/scala/org.apache.spark.sql.connect.client/Dataset.scala`

Reply:
Argh, thought Intellij would automatically expand here